climpred/tests/test_predictability_horizon.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pytest
+import xarray as xr
+
+from climpred.predictability_horizon import _last_item_cond_true, predictability_horizon
+
+
+@pytest.mark.parametrize("threshold,expected", [(1, 1), (2, 3), (3, 6), (3.5, 6)])
+def test_least_item_cond_true(threshold, expected):
```

Comment:
This is where you could test disjoint coordinate space for example. Or cases where the lead time coordinate starts with 0 like it does for some SubX style runs. Then also check the case where all are True and the case where the first is False.